mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/logging/impl/Log.java

```diff
@@ -322,4 +328,10 @@ SearchException invalidFieldEncodingForKeywordFieldMapping(StandardIndexFieldTyp
 	)
 	SearchException invalidContainerExtractorReferencingBothBuiltinExtractorAndExplicitType(@FormatWith(EnumFormatter.class) BuiltinContainerExtractor value,
 			@FormatWith(ClassFormatter.class) Class<? extends ContainerExtractor> type);
+
+	@Message(id = ID_OFFSET_2 + 43, value = "Error on mapping back URI '%1$s' to URL.")
```

Comment:
Ok, I understand you did what I asked. But my comment made sense on this specific commit. Since a later commit changes the bridge in such a way that we don't convert between URI and URL, the message I gave does not make sense anymore. Maybe you could have adapted the message accordingly... ?

I'll do it.